Rakefile

```
@@ -17,3 +17,18 @@ task :test do
 end
 
 task default: :test
+
+task :get_xkcd do
+  xkcd = GetColors::XKCD.new
+  xkcd.get_data
+  xkcd.dump_file("lib/colors/color_data/xkcd.rb")
+end
+
+task :get_css4 do
+  css4 = GetColors::CSS4.new
+  css4.get_data
+  css4.dump_file("lib/colors/color_data/css4.rb")
+end
+
+desc "Get color data files from the web, per issue #8"
+task :get_colors => [:get_xkcd, :get_css4]
```

How about `colors:generate`?

```suggestion
namespace :colors do
  namespace :generate do
    task :xkcd do
      # ...
    end
    task :css4 do
      # ...
    end
  end
  task :generate => [:xkcd, :css4]
end
```